Shuttlerock/actions
# [SECURITY-228] Prioritize pull requests labeled 'security', and send a Slack message to the owner

## Prioritize pull requests labeled 'security', and send a Slack message to the owner

[Jira Task](https://shuttlerock.atlassian.net/browse/SECURITY-228)
[Jira Epic](https://shuttlerock.atlassian.net/browse/SECURITY-244)

We should hook into the &#x60;pull-request-labeled-action&#x60;action in the automation. If the PR is labeled with &#x60;security&#x60;, we should:

1. Add a default priority label
1. Sync the ‘security’ label with the Jira issue
1. Ping the owner of the PR in Slack asking them to confirm the priority.

## How to test the PR

Add the `security` label to a PR, and check that a priority is assigned, and a slack message is sent.

## Deployment Notes

None
